src/Qt/AudioPlaybackThread.cpp

```diff
@@ -30,15 +31,18 @@ namespace openshot
 	// Global reference to device manager
 	AudioDeviceManagerSingleton *AudioDeviceManagerSingleton::m_pInstance = NULL;
 
-    // Create or Get audio device singleton with default settings (44100, 2)
-    AudioDeviceManagerSingleton *AudioDeviceManagerSingleton::Instance()
-    {
-        return AudioDeviceManagerSingleton::Instance(44100, 2);
-    }
+	// Create or Get audio device singleton with default settings (44100, 2)
+	AudioDeviceManagerSingleton *AudioDeviceManagerSingleton::Instance()
+	{
+		return AudioDeviceManagerSingleton::Instance(44100, 2);
+	}
 
 	// Create or Get an instance of the device manager singleton (with custom sample rate & channels)
 	AudioDeviceManagerSingleton *AudioDeviceManagerSingleton::Instance(int rate, int channels)
 	{
+		static std::mutex mutex;
```

Comment:
Adding mutex to prevent concurrent calls to the `Instance()` method - which seemed to be happening during debugging